### PR TITLE
Reorg

### PR DESCRIPTION
**Pull request by torbsorb**
This pull request contains a reorganization based on https://zividlabs.atlassian.net/wiki/spaces/CS/pages/758251542/Samples+complete+re-org+1.7.
The first commit is simply reorg of existing samples
The second commit includes samples from SDK into github

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ project(CppExtraSamples)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 include(CompilerOptions)
 
-set(ZIVID_VERSION 1.7.0)
+set(ZIVID_VERSION 1.8.0)
```
Comment (torbsorb) [inline CMakeLists.txt]: @nedrebo, @SatjaSivcev 
This line is causing build to fail right now. Should I just remove it or will it get updated soon?

Comment (nedrebo) [reply CMakeLists.txt]: The reason this fails is that the CI setup scrips must be updated to install 1.8.0 as well. I would do the bump to 1.8.0 in a separate PR to keep PRs and commits simple and separated.

However, the Zivid SDK is backwards compatible. Unless the samples use functionality introduced in 1.8.0 it is not necessary to bump the required version here.

File: Applications/Advanced/HandEyeCalibration/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+## Hand Eye Calibration
```
Comment (nedrebo) [inline Applications/Advanced/HandEyeCalibration/README.md]: Top level heading should only have one #

I use the markdownlint plugin in VS code to help with consistent markup use and formatting

File: CMakeLists.txt [hunk 3]
```diff
@@ -12,22 +12,15 @@ option(USE_OPENCV "Enable samples which depend on OpenCV" ON)
 option(USE_PCL "Enable samples which depend on Point Cloud Library (PCL)" ON)
 
 set(SAMPLES
-    Zivid/CaptureHDRCompleteSettings
```
Comment (nedrebo) [inline CMakeLists.txt]: Nice getting rid of all the redundant Zivids.

File: README.md [hunk 4]
```diff
@@ -4,46 +4,57 @@ This repository contains additional **C++** code samples for **Zivid**.
 
 [![Build Status][ci-badge]][ci-url]
 
-The basic samples are available at [https://www.zivid.com/downloads](https://www.zivid.com/downloads).
-The Windows Zivid installer adds these samples in C:\Users\Public\Documents\Zivid\samples and they should build out of the box using Visual Studio 2015 or 2017.
 Check out our [tutorial on configuring and building these samples on Ubuntu](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/59441336/Configure+C+Samples+with+CMake+and+then+build+them+using+make+in+Ubuntu).
```
Comment (nedrebo) [inline README.md]: For better readability of readme in plain text mode you can add urls to the end of the file like seen above. If you agree that is an improvement..

File: README.md [hunk 5]
```diff
@@ -1,49 +1,71 @@
...
+			- **Dependencies:**
+				- [Eigen](http://eigen.tuxfamily.org/) version 3.3.7 or newer
+		- [**CaptureUndistortRGB**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/CaptureUndistortRGB/CaptureUndistortRGB.cpp) - Use Zivid camera intrinsics to undistort an RGB image. This example will prompt the user for whether to capture a 2D or a 3D image. In both instances it will operate on a 2D image. However, in the 3D case it will extract 2D image from a ZDF point cloud. The 2D variant is faster.
+			- **Dependencies:**
+				- [OpenCV](https://opencv.org/) version 4.0.1 or newer
+		- [**CreateDepthMap**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp) - Import a ZDF point cloud and convert it to OpenCV format, then extract and visualize depth map.
+			- **Dependencies:**
+				- [OpenCV](https://opencv.org/) version 4.0.1 or newer
+
+## Samples to be removed
```
Comment (nedrebo) [inline README.md]: I don't understand this heading...

File: README.md [hunk 6]
```diff
@@ -1,49 +1,71 @@
...
-	- [**ReadPCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/ReadPCLVis3D/ReadPCLVis3D.cpp) - This example shows how to read a PCL point cloud and visualize it.
-	- [**CaptureWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud, save it to a .PCD file format, and visualize it.
 	- [**CaptureFromFileWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureFromFileWritePCLVis3D/CaptureFromFileWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud from an emulated Zivid camera, save it to a .PCD file format, and visualize it.
 	- **Dependencies:**
 		- [PCL](http://pointclouds.org/) version 1.9.1
 
-- [**Downsample**](https://github.com/zivid/cpp-extra-samples/tree/master/Downsample)  - This example shows how to import a Zivid point cloud from a .ZDF file and downsample it.
-	- **Dependencies:**
-		- [Eigen](http://eigen.tuxfamily.org/) version 3.3.7 or newer
+## Samples renamed
```
Comment (nedrebo) [inline README.md]: ...and this.

File: CMakeLists.txt (hunk 1 above)
Comment (nedrebo) [reply CMakeLists.txt]: I see that you did the change in a separate commit, that is probably sufficient. No need for separate PR.

Comment (torbsorb) [reply CMakeLists.txt]: Is it possible to make CI run on the previous commit and ignore this until it's updated?

File: README.md (hunk 5 above)
Comment (torbsorb) [reply README.md]: It's for your eyes only :) I guess the fact that these are deleted will appear in a git commit

File: README.md (hunk 6 above)
Comment (torbsorb) [reply README.md]: It's for your eyes only :) I guess the fact that these are renamed will appear in a git commit

File: CMakeLists.txt (hunk 1 above)
Comment (nedrebo) [reply CMakeLists.txt]: Yes, just don't do this change.

**Comment by nedrebo** [comment]
No more comments

**Pull request merged**